NewBooks+Course.md

```diff
@@ -0,0 +1,6 @@
+
+* [Ajax for beginners](http://www.academiepro.com/uploads/livres/AjaxProgramming.pdf)
+
+* [GameMaker Language](https://ia601503.us.archive.org/22/items/game-maker-game-programming-with-gml-learn-game-maker-language-programming-conce/GameMaker%20game%20programming%20with%20GML%20_%20learn%20GameMaker%20language%20programming%20concepts%20and%20script%20integration%20with%20GameMaker_%20Studio%20through%20hands-on%2C%20playable%20examples.pdf)
+
+* [Mobile AppDev Harvard cs50](https://cs50.harvard.edu/mobile/2018/)
```

Comment:
Hello, 

- the Ajax resource and GameMaker resource belong [here](https://github.com/EbookFoundation/free-programming-books/blob/master/free-programming-books.md)

- the cs50 resources belongs [here](https://github.com/EbookFoundation/free-programming-books/blob/master/free-courses-en.md) 

- Lastly, your resources need to have " (PDF)" added to it at the end. Refer to [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)